### PR TITLE
Prepare for pg v16.4 and fix symbol extraction

### PR DESCRIPTION
**Pull request by pmp-p**

### CODE REVIEW

File: cibuild.sh [hunk 1]
```diff
@@ -368,6 +450,22 @@ do
             du -hs ${WEBROOT}/*
         ;;
 
+        pglite-test) echo "================== pglite-test ========================="
```
Comment (msfstef) [inline cibuild.sh:453]: is there a reason this is added as a task in cibuild.sh? This is already part of the workflow within `packages/pglite`?

**Review by msfstef — APPROVED** [review]
Looks good!

File: cibuild.sh (hunk 1 above)
Comment (pmp-p) [reply cibuild.sh:453]: yes it is an option for a streamlined CI with no cache which can publish to web without passing all the tests ( eg for debugging purpose here https://github.com/pmp-p/pglite-16.x/blob/main/.github/workflows/CI.yml and https://pmp-p.github.io/pglite-16.x/postgres.html )

**Comment by pmp-p** [comment]
finally all green (after 4 retries) !

**Pull request merged**